### DISCUSSION TITLE
Print / Export to PDF styles

### DISCUSSION THREAD

**Discussion by arthursoares** [jdanielmourao/obsidian-sanctum#101] [Ideas]
First of all, congrats on the theme. It has become my default.

I would love to suggest some additional styling for exporting to PDF / prepare for printing. A couple of ideas or references:

- Removing the [gradient from the top](https://user-images.githubusercontent.com/533339/147960805-ed0715e7-9073-4aeb-ae43-45e8ea41f41d.png) of the first page 
- Adjusting the page `max width`.
- Printer friendly colors (when using the themes and highlights)
- Support for `aside` element (it usually cuts out of the margins)

etc…

**Update:** I just saw this entry on the roadmap: https://github.com/users/jdanielmourao/projects/1/views/4

**Comment by jdanielmourao** [comment]
Hi! Yeah, I'm slowly editing print, but some things are a bit hard to do the way Print is currently setup in obsidian. I've managed to include side notes, for example, but text keep getting cropped at the end of the page, amongst other things that don't obey regular css printing norms.
Here's a preview, in any case:
![image](https://user-images.githubusercontent.com/91087143/147961969-1e8484d1-9f50-4c14-a404-7d7b56620403.png)

**Reply by arthursoares** [reply]
Yes! I've even tried using that LaTeX plugin, but not much success. 

I've been compiling some interesting resources that might be helpful:

- https://www.vojtechruzicka.com/css-printing/ : Describes great tips for pagination / link handling
- https://alistapart.com/article/improvingprint/ : Link display

**Reply by arthursoares** [reply]
And this might help debugging: https://github.com/nothingislost/obsidian-print-preview

**Reply by jdanielmourao** [reply]
Yeah, I have been using Obsidian Print Preview, as well as those resources, or similar ones. Thing is some of that CSS, like margins for example, don't work all that well, or intuitively.

**Reply by jdanielmourao** [reply]
I can't use `@page`, for example
